src/lib.rs

```diff
@@ -11,8 +11,8 @@
 //!     std::env::var("PASSAGE_API_KEY").unwrap(),
 //! );
 //!
-//! let app_info = passage_flex.get_app().await.unwrap();
-//! println!("{}", app_info.auth_origin);
+//! let passage_user = passage_flex.user.get(external_id).await.unwrap();
+//! println!("{:?}", passage_user.webauthn_devices);
```

Comment:
It might be simpler to remove this part of the usage example. The previous method was clear on its own, but most flex methods need context (e.g., what’s an `external_id`?).

Reply:
removed in [b7160f9](https://github.com/passageidentity/passage-flex-rust/pull/30/commits/b7160f9d857f65d0fbf587901f04b07c68ba74a5)